crowd_anki/config/config_dialog.py

```diff
@@ -0,0 +1,75 @@
+"""
+Config dialog entry point.
+This module will load the config initially.
+Write actions will be triggered here when the user clicks the dialog "Save All" button.
+"""
+
+from aqt.qt import *
+
+from .config_ui import Ui_Dialog as ConfigUI
+from .config_settings import ConfigSettings
+
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+class ConfigDialog(QDialog):
+    def __init__(self, parent=None):
+        super().__init__()
+        self.parent = parent
+        self.config = ConfigSettings()
+        self.form = ConfigUI()
+        self.form.setupUi(self)
+        self.ui_initial_setup()
```

Comment:
`setup_from_config`?

Reply:
Are you suggesting to change the `ui_initial_setup` function to `setup_from_config`? The current name makes more sense to me: it sets up the initial values and onchange providers of all the UI parts.